docs/examples/api-reference/client.py

```diff
@@ -41,7 +41,7 @@ class UserFeatures:
     name: str
     country_geoid: int
     # The users gender among male/female/non-binary
-    age: int = F().meta(owner="aditya@fennel.ai")
+    age: int = feature().meta(owner="aditya@fennel.ai")
```

Comment:
I think there is a place in docs where we suggest this as a common shorthand to use - could you plz change/remove that too? https://fennel.ai/docs/concepts/featureset#convention